README.md

```diff
@@ -1,7 +1,6 @@
-<img src="https://github.com/habitat-sh/habitat/blob/master/www/source/images/habitat-logo.png" width="200">
+<img src="images/chef-logo.svg">
```

Comment:
I would remove this entirely. I renders really large but none of our other product repos have a logo.